bait-files/src/test/java/com/google/sps/UsernameTakenTest.java

```diff
@@ -27,11 +27,11 @@
 import org.junit.runners.JUnit4;
 
 @RunWith(JUnit4.class)
-public final class UserTest {
+public final class UsernameTakenTest {
```

Comment:
Nit: Include the word "Servlet" in this class and file name: UsernameTakenServletTest.  To make it super obvious that the class you are testing is UsernameTakenServlet.

Reply:
Rename to servlet